internal/pkg/deploy/cloudformation/stack/env.go

```diff
@@ -61,6 +61,11 @@ type envReadParser interface {
 	ParseEnvBootstrap(data *template.EnvOpts, options ...template.ParseOption) (*template.Content, error)
 }
 
+type addons struct {
+	URL    string
+	Config NestedStackConfigurer
```

Comment:
1. If clients will populate these fields can we add doc comments 🥺 
2. What do you think of these alternative names?
```suggestion
	S3ObjectURL    string
	Stack NestedStackConfigurer
```